src/Models/HydrostaticFreeSurfaceModels/split_explicit_free_surface.jl

```diff
@@ -95,8 +95,8 @@ SplitExplicitAuxiliary{𝒞ℱ, ℱ𝒞}
 # Members
 `Gᵁ` : (ReducedField). Vertically integrated slow barotropic forcing function for U
 `Gⱽ` : (ReducedField). Vertically integrated slow barotropic forcing function for V
-`Hᶠᶜ`: (ReducedField). Depth at (Face, Center)
-`Hᶜᶠ`: (ReducedField). Depth at (Center, Face)
+`Hᶠᶜ`: (ReducedField). Depth at (Face, Center): minimum depth of neighbors
+`Hᶜᶠ`: (ReducedField). Depth at (Center, Face): minimum depth of neighbors
```

Comment:
The comment is slightly misleading; this is only the case for some types of immersed boundaries